diagnostic.md

```diff
@@ -51,13 +86,31 @@ Write the `curl` request you'd use to delete a single donut, using whatever ID
 you'd like.
 
 ```sh
-# your answer here
+curl --request DELETE http://www.example.com/donuts/12
+
 ```
 
 Write an AJAX request to delete a single donut from the server.
 
 ```js
-let deleteDonut = /* your answer here */;
+let deleteDonut = function deleteDonut() {
+    debugger;
+    $.ajax({
+        type: "DELETE",
+        url: "http://www.example.com/donuts",
```

Comment:
Wrong URL - we're trying to delete a single donut, so we need to specify in the URL which donut we want to delete.